packages/utils/test/ratelimit.test.ts

```diff
@@ -197,3 +197,35 @@ describe('updateRateLimits()', () => {
     expect(updatedRateLimits.all).toEqual(60_000);
   });
 });
+
+describe('data category "metric_bucket"', () => {
```

Comment:
Let's add another test case for `2700:metric_bucket:organization:quota_exceeded:`